css/properties/vector-effect.json

```
@@ -10,12 +10,12 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

```suggestion
              "version_added": "6"
```

See: https://github.com/caugner/css-properties/commit/cec7aa0579febcec20c075cf604e4a37e85e4437#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R200